### PR TITLE
[HTP-0004] app-nexus: add original app-nexus bidder with ccpa support

### PR DESCRIPTION
**Pull request by jsnellbaker**
## Type of Change

- [x] New adapter
- [x] Feature Update

## Description of Change
Per the request made [here](https://github.com/indexexchange/ix-library-adapters/pull/52#discussion_r360434835), I have created a new PR using the original version of our app-nexus bidder in order to add CCPA support.

Please let me know if anything further is needed.

### CODE REVIEW

File: app-nexus/app-nexus-htb.js [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+                },
+                rateLimiting: {
+                    enabled: false,
+                    value: 0
+                }
+            },
+            targetingKeys: {
+                id: 'ix_apnx_id',
+                om: 'ix_apnx_om',
+                pm: 'ix_apnx_dealid'
```
Comment (ix-certification) [inline app-nexus/app-nexus-htb.js]: This `pm` value was changed from the original app-nexus-adapter code. Originally, it was set to `ix_apnx_pm`. Any reason for this change? Keep in mind that changing this property will have an effect on lines setup with old pm key.

**Comment by ix-certification** [comment]
`app-nexus/` is in the .eslintignore file. Please remove this to ensure from the file.

File: app-nexus/app-nexus-htb.js (hunk 1 above)
Comment (jsnellbaker) [reply app-nexus/app-nexus-htb.js]: I have pushed a change to revert this key.  It should now be back to `ix_apnx_pm`.

**Comment by jsnellbaker** [comment]
@ix-certification I have made the requested update to the `.eslintignore` file and a subsequent lint error that was found.  Please take another look when you have the chance.  Thanks.

**Pull request merged**